app/assets/stylesheets/components/_progress-indicator.scss

```diff
@@ -1,7 +1,6 @@
-
 .progress-indicator {
   width: 345px;
-  margin: -25px 0 25px 0;
+  margin: 0 0 25px 0;
```

Comment:
We're making up for the negative margin by adding the "back" button above it and then giving that button a bottom margin.